draft-ietf-quic-recovery.md

```diff
@@ -1480,8 +1480,9 @@ GetLossTimeAndSpace():
 GetPtoTimeAndSpace():
   duration = (smoothed_rtt + max(4 * rttvar, kGranularity))
       * (2 ^ pto_count)
-  // Arm PTO from now when there are no inflight packets.
-  if (no in-flight packets):
+  // Arm PTO from now when there are no ACK-eliciting packets in
+  // flight.
```

Comment:
```suggestion
  // Arm PTO when there are no ACK-eliciting packets in flight.
```

Reply:
I like dropping a line, but I think the existing text is slightly clearer, since this is the only time we set PTO from now().

Reply:
Sure, how about this then...

> // Arm PTO if no ACK-eliciting packets are in flight.

Reply:
(Whoops, github didn't display your response at first...)

I actually found the original wording difficult to parse as a well-formed sentence. "From now when" is weird. We could rephrase it as "based on the current time" or something but that's getting wordy, and I don't feel like it's really adding anything over the psuedocode.

Reply:
Made another pass at the wording here.